.github/workflows/minimum-rust-version.yml

```diff
@@ -54,4 +47,4 @@ jobs:
           key: msrv-${{ matrix.settings.target }}-cargo-${{ matrix.settings.os }}
 
       - name: Cargo check MSRV
-        run: cargo check --all-features
+        run: cargo +"${{ steps.toolchain.outputs.RUST_TOOLCHAIN }}" check --all-features
```

Comment:
Are we not using the MSRV version here?

Reply:
On the lint workflow the `rust-toolchain` file was overriding the installed version, which was causing the errors we saw (we installed 1.75 but cargo tried to use 1.85, which wasn't installed)

I can't say for sure for sure if the same thing is happening here, but since this is the only other place were we want to override the rust version used, I figured might as well be safe and explicitly select the version.

Reply:
Ah, we read toolchain version from cargo.toml.